bundle/run.sh

```diff
@@ -247,6 +247,7 @@ Dial:
 JwksURL: ${JWKS_URL}
 ResourceDirectoryAddr: ${RESOURCE_DIRECTORY_ADDRESS}
 CertificateAuthorityAddr: ${CERTIFICATE_AUTHORITY_ADDRESS}
+WWWDirectory: /usr/local/var/www
```

Comment:
This should be the default value in the http-gw service.

Reply:
Fixed